CHANGELOG.md

```diff
@@ -4,6 +4,24 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 
+## Release 10.0.0 (2020-10-15)
```

Comment:
This needs to go into the [`RELEASENOTES.md` ](https://github.com/realm/realm-studio/blob/channel/beta/RELEASENOTES.md).

Reply:
Updated (incl. date)